src/kibana/styles/main.less

```diff
@@ -203,7 +203,16 @@ notifications {
     z-index: 2 !important;
   }
 }
-
+.fa-search-less-than-equal:before {
+  content: "\2264";
```

Comment:
awesome approach! I would name them something other than 'fa' cause they aren't part of font awesome. like 
'lr-custom--search-less-than-equal' or something